migrate/scripts/object-id.js

```diff
@@ -0,0 +1,42 @@
+var mongodb = require("mongodb"),
+    ObjectId = mongodb.ObjectId
+
+const canRun = collection => (
+  new Promise((resolve, reject) => {
+    collection.find({
+      '_id': {
+        $type: "string"
+      }
+    })
+    .toArray()
+    .then(r => {
+      const ids = r.map(e => e._id);
+
+      resolve(ids.some(i => i));
```

Comment:
How about `resolve(ids.some(i => i._id))` for being concise?  No worries if you don't like it.